generator/integration_tests/test.proto

```diff
@@ -63,6 +63,14 @@ service GoldenThingAdmin {
     option (google.api.method_signature) = "parent";
   }
 
+  // Does Nothing.
```

Comment:
if this was the las method you would have fewer spurious changes due to line numbers...  Just saying.

Reply:
You're right. I moved it into the last `service`.
(less overall changes, but half the line numbers changed in the latest commit 🤷‍♂️)